test/environmentSettings.ts

```diff
@@ -48,7 +50,7 @@ export class EnvironmentSettings {
   private _throttleFactor = 1;
   private _javaHome = process.env.JAVA_HOME;
   private _useExistingProject: string | undefined;
-  private _logLevel: LogLevel = 'info';
+  private _logLevel: LogLevel = 'debug';
```

Comment:
Given the change in wdio.config.ts, it seems this should be `warn`

Reply:
I have an item in the PL to talk about this because I wasn't sure about some stuff about loglevel